addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -3,81 +3,126 @@
 
 <template id="index" name="Jobs">
     <t t-call="website.layout">
-        <div id="wrap" class="o_website_hr_recruitment_jobs_list bg-100">
-            <div class="oe_structure">
-                <section class="pt48 pb32 bg-200">
-                    <div class="container">
-                        <div class="row">
-                            <div class="col-lg-12 text-center">
-                                <h2 class="h1 text-secondary">Our Job Offers</h2>
-                                <h3 class="text-muted">Join us and help disrupt the enterprise market!</h3>
-                                <p>
-                                    Join us, we offer you an extraordinary chance to learn, to
-                                    develop and to be part of an exciting experience and
-                                    team.
-                                </p>
-                            </div>
-                        </div>
+        <div id="wrap" class="o_website_hr_recruitment_jobs_list o_cc o_cc2">
```

Comment:
I am not sure it's a good idea to use o_cc2 on a main container (especially with the missing o_colored_level) class... but maybe. In any case, I think there should be some uniformization accross the apps (cc @stefanorigano) to handle this. In events, it's white with blue cards, in slides the body color is darkened, here we now use o_cc2, ... and I am sure all the other apps have done it differently too.

I'll leave like this for now (just adding the o_colored_level class (which I'd like to get rid of at some points if possible...)).